pkg/api/datasources.go

```diff
@@ -331,6 +332,20 @@ func validateURL(cmdType string, url string) response.Response {
 	return nil
 }
 
+func isCacheHit(r caching.CachedResourceDataResponse) bool {
```

Comment:
I could also make this a method on CachedResourceDataResponse, would that be better?